### ISSUE TITLE
Available languages should not be translated

### ISSUE THREAD

**Issue by Spomky** [lunetics/LocaleBundle#144]
I added your bundle in my project and it works perfectly.
I noted that languages displayed in the switcher are translated into the current language.

* английский
* немецкий
* русский
* японский

Guess which one is the best for you! 

I really think that languages should be displayed into its own language, so the default template of the switcher should use `locale_target_language` instread of `locale_current_language`.

* English
* Deutsch
* Français
* русский
* 日本語

**Comment by borNfreee** [comment]
It would be greatful to have an option to control this behaviour.

**Comment by dbu** [comment]
this is more or less prepared in [getTargetInformations](
https://github.com/lunetics/LocaleBundle/blob/master/Switcher/TargetInformationBuilder.php#L91). but the [template is using locale_current_language](https://github.com/lunetics/LocaleBundle/blob/master/Resources/views/Switcher/switcher_links.html.twig). i think what would make most sense is to provide another template and document to configure that other template to get the languages in locale_target_language.

**Comment by Spomky** [comment]
Yes this is easy to integrate because informations are correctly sent to the template.
Another template could be a good idea, but what I mean is that this new template should be the default one.

```twig
{# Resources/views/Switcher/targer_languages_switcher_links.html.twig #}
<p>Switch to :</p>
<ul class="locale_chooser">
    {% for locale in locales %}
        <li{% if current_locale == locale.locale %} class="locale_current"{% endif %}>
            <a href="{{ locale.link }}" title="{{ locale.locale_target_language }}">{{ locale.locale_target_language }}</a>
        </li>
    {% endfor %}
</ul>
```

**Comment by dbu** [comment]
IMO we should just provide that template (what you propose looks correct
to me) and document things.

changing the default is a BC break and could surprise the users. we
should only do that when we bump to a version 2.0 of this bundle.

**Comment by ikwattro** [comment]
It is a long time ago I wrote the switcher. 

IMO every website will need a different template as well for design purposes than for the locale display, that's why I provided the both possibilities as template variables https://github.com/lunetics/LocaleBundle/blob/master/Switcher/TargetInformationBuilder.php#L121

In fact, the template available could be empty and you would need to override it, Symfony makes it very easy to override the template. I don't see a reason to provide multiple languages in the library.
